.github/workflows/ci.yml

```diff
@@ -233,3 +233,92 @@ jobs:
         cargo fuzz run block_roundtrip -- -max_total_time=20
         cargo fuzz run chainman_process_block -- -max_total_time=20
         cargo fuzz run script_verify -- -max_total_time=20
+
+  fuzz-corpus:
```

Comment:
I think we can just completely replace the existing fuzzing job with this one. Can you also activate the address sanitizer for this job?

Reply:
Cool! Removed the fuzz setup test in the next commit and added the sanitizer for the new. Thanks for the feedback.